### PR TITLE
Tools: update eslint, stylelint, prettier & babel + optional chaining

### PR DESCRIPTION
**Pull request by Drapegnik**
- https://github.com/tc39/proposal-optional-chaining#overview-and-motivation
- https://babeljs.io/docs/en/babel-plugin-proposal-optional-chaining

### CODE REVIEW

**Comment by Drapegnik** [comment]
[optional chain operator](https://trello.com/c/NHKvyEoV/458-optional-chain-operator)

File: utils/validators.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ export const hasErrors = (errors, touched) =>
   }).length;
 
 const emailRegexp = /^(([^<>()[\].,;:\s@"]+(\.[^<>()[\].,;:\s@"]+)*)|(".+"))@(([^<>()[\].,;:\s@"]+\.)+[^<>()[\].,;:\s@"]{2,})$/;
-export const isEmail = email => !!email && email.match(emailRegexp);
+export const isEmail = email => !!email?.match(emailRegexp);
```
Comment (UladBohdan) [inline utils/validators.js:16]: do we need to keep `!!` here?

Comment (Drapegnik) [reply utils/validators.js:16]: yep, 
```
email?.match // -> undefined, in case of email absence 
```

Comment (UladBohdan) [reply utils/validators.js:16]: so `!!` applied in the very end?
`!!undefined = false` ?

**Comment by Drapegnik** [comment]
[prettier@2](https://trello.com/c/4kcw8lJX/498-prettier2)

**Review by UladBohdan — APPROVED** [review]

**Pull request merged**